openlibrary/plugins/openlibrary/js/add-book.js

```diff
@@ -72,6 +76,8 @@ function clearErrors() {
     errorDiv.classList.add('hidden');
     const confirm = document.getElementById('confirm-add');
     confirm.classList.add('hidden');
+    const foundDiv = document.getElementById('id-found');
+    foundDiv.classList.add('hidden');
```

Comment:
```suggestion
```
There is already a div for displaying ID errors, which should be used for this as well.